frontend/src/Buildings/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,97 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Buildings /> Should render buildings and match the snapshot 1`] = `
+<View
+  style={
+    Object {
+      "alignItems": "center",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
+>
+  <CardView
+    style={
+      Object {
+        "flexDirection": "row",
+      }
+    }
+  >
+    <Unknown
```

Comment:
do you know why are these `Unknown`? just curious

Reply:
I think it's because I manually mocked the subcomponent here instead of shallow

Reply:
thanks!